web/package.json

```diff
@@ -2,8 +2,9 @@
   "name": "client",
   "version": "0.1.0",
   "private": true,
-  "proxy": "https://parity.ocua.ca",
+  "proxy": "http://localhost:5000",
```

Comment:
this default used to flip base on convenience but now that the client is not "read only" it should always point to the local env.